### ISSUE TITLE
Simple example not working - error in console

### ISSUE THREAD

**Issue by debs-obrien** [nuxt-community/storybook#25]
### Version
@nuxtjs/storybook: v1.0.0
nuxt: v2.14.0

### Reproduction Link
https://github.com/nuxt/nuxtjs.org/tree/test-storybook

### Steps to reproduce
run the dev command for storybook and check console

### What is Expected?
should show stories. followed example as per docs with Link.vue and Link.stories.js. Also updated all packages with fresh install etc just incase but still get error.

### What is actually happening?
page is not loading properly
<img width="455" alt="Screenshot 2020-07-31 at 16 06 07" src="https://user-images.githubusercontent.com/13063165/89042783-c33e4c00-d347-11ea-865a-b9eab25c4892.png">

<img width="555" alt="Screenshot 2020-07-31 at 15 54 29" src="https://user-images.githubusercontent.com/13063165/89042698-a43fba00-d347-11ea-8250-d72eacea185c.png">

**Comment by Tchoupinax** [comment]
Hello !

I have exactly the same issue. Page is not loading.
![image](https://user-images.githubusercontent.com/9136206/89043462-cede4280-d348-11ea-9cce-77f9077997f5.png)

Error is at line 23 (I change my personal path)
![image](https://user-images.githubusercontent.com/9136206/89043587-fc2af080-d348-11ea-8ad4-e4d6c7ac2b7e.png)

I'm using `nuxt 2.14.0` and i also have activated `components: true` options to have global components. :)

**Comment by pi0** [comment]
Issue is that we are not applying custom entry points and calling `webpack:config` (because actual nuxt build is mocked). @farnabaz is working on it 😊

**Comment by altryne** [comment]
Have the same issue, pasting the error here in plain text so searches can take folks to this issue
```Uncaught ReferenceError: components is not defined
    at Module../.nuxt-storybook/storybook/preview.js (main.6f5e15f07784cf4afe03.bundle.js:565)
    at __webpack_require__ (runtime~main.6f5e15f07784cf4afe03.bundle.js:849)
    at fn (runtime~main.6f5e15f07784cf4afe03.bundle.js:151)
    at Object.0 (main.6f5e15f07784cf4afe03.bundle.js:19403)
    at __webpack_require__ (runtime~main.6f5e15f07784cf4afe03.bundle.js:849)
    at checkDeferredModules (runtime~main.6f5e15f07784cf4afe03.bundle.js:46)
    at Array.webpackJsonpCallback [as push] (runtime~main.6f5e15f07784cf4afe03.bundle.js:33)
    at main.6f5e15f07784cf4afe03.bundle.js:1```

**Comment by chrisleyva** [comment]
@altryne I have the exact same issue. Thanks.

**Comment by farnabaz** [comment]
Fixed in [v1.1.0](https://github.com/nuxt-community/storybook/releases/tag/v1.1.0)
